install

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+#TODO(by black7375)
+#However, there may be existing user's settings, so back up and make a copy.
+#I just made a simple one yesterday, assuming that there may be multiple backup states.
+#I think it's better to do a redirection(>>) after backup to merge the contents of the file.
+printf "Are you using ESR, Default, or Dev?(ESR/Default/Dev): "
+read -r
+if [ "$REPLY" == "ESR" ] || [ "$REPLY" == "esr" ]; then
+  profiledir="$(grep '.default-esr' ~/.mozilla/firefox/profiles.ini | grep 'Default=' | cut -f 2 -d'=')"
+elif [ "$REPLY" == "Default" ] || [ "$REPLY" == "default" ]; then
+  profiledir="$(grep '.default-release' ~/.mozilla/firefox/profiles.ini | grep 'Default=' | cut -f 2 -d'=')"
+elif [ "$REPLY" == "Dev" ] || [ "$REPLY" == "dev" ]; then
+  profiledir="$(grep '.dev-edition-default' ~/.mozilla/firefox/profiles.ini | grep 'Default=' | cut -f 2 -d'=')"
+else
+  echo "Unspecified."
+  exit
+fi
+if [[ -f ~/.mozilla/firefox/${profiledir}/user.js ]]; then
+    printf "user.js exists. Do you want to make a backup of it?(Y/n): "
+    read -r
+    if [ "N" == "$REPLY" ] || [ "n" == "$REPLY" ]; then
+      printf "Overwriting...\n"
+      rm -rf ~/.mozilla/firefox/"${profiledir}"/user.js
+      cp user.js ~/.mozilla/firefox/"${profiledir}"/user.js
+    else
+      printf "Making a backup...\n"
+      if [[ -d "$HOME/.mozilla/firefox/${profiledir}/backup" ]]; then
+        rm -rf ~/.mozilla/firefox/"${profiledir}"/backup
+      fi
+      mkdir ~/.mozilla/firefox/"${profiledir}"/backup/
+      mv ~/.mozilla/firefox/"${profiledir}"/user.js ~/.mozilla/firefox/"${profiledir}"/backup/user.js
```

Comment:
Is it necessary to create a backup directory?
 I prefer `${FILENAME}.bak`

Refer: https://en.wikipedia.org/wiki/Bak_file

Reply:
Hm.. ok. But using .bak won't be good since copying those files to a backup folder could be better for the user since copying the files could be easier in the file explorer.